### ISSUE TITLE
XAMPP running as `daemon` user instead of `www-data`

### ISSUE THREAD

**Issue by PotatoCider** [tomsik68/docker-xampp#54]
# Bug Report

I'm running Docker on: **Linux**

## XAMPP start command

I used the following command to start the xampp container:

```bash
docker run --name xampp -p 41061:22 -p 80:80 -d -v ~/my_web_pages:/opt/lampp/htdocs/www tomsik68/xampp:7
```

I mounted a Wordpress website, but could not upload files into the `wp-content/` directory (permissions: `744`, owner: `www-data`).

## Additional steps to trigger the bug

After starting the container, I did the following:

1. Get a bash shell in the container: `docker exec -it xampp bash`
2. Run `top`
3. Process `/opt/lampp/bin/` is running as `daemon` user

![image](https://github.com/tomsik68/docker-xampp/assets/28196972/a6fd5148-3dc7-418f-9257-4f1fef07eb41)

## Expected behavior

XAMPP should run as `www-data` (UID: 33)

## Actual behavior

XAMPP is running as user `daemon` (UID: 1)

## Note

I'm not sure if this is intended behaviour. But it seems that the `daemon` user and group is only used for legacy purposes.

Also, changing the owner of the `wp-content/` folder to `daemon` fixes the file upload.

**Comment by tomsik68** [comment]
Hey, good question thanks for bringing this up! I'm also not 100% sure about the intended behavior so I did some digging:

1. I tried adding a `USER www-data` right before the final CMD which caused the command to be executed by the `www-data`. This was the output for latest version of xampp:

```
Starting XAMPP for Linux 8.2.4-0...
You need to be root to perform this action.
```

This shows that the script is intended to be started as root. 

2. The script calls `apachectl` which, in turn, calls `httpd -k start`. Using `strace` on it reveals that it daemonizes itself in a way that it uses UID 1.

My take on this situation is that using UID 1 is a "feature" of XAMPP. In this case, I think that is the expected behavior. I agree it is surprising but I also don't see an easy way how to fix this at the level of Dockerfile without turning this into my own XAMPP "distribution". Therefore, I won't make any changes to this behavior.

I'll leave this open for a little while in case there are any additional questions
